cmd/tether/main_linux.go

```diff
@@ -38,6 +38,10 @@ func main() {
 		halt()
 	}()
 
+	logFile, _ := os.OpenFile("/dev/ttyS1", os.O_WRONLY|os.O_SYNC, 0644)
+	syscall.Dup2(int(logFile.Fd()), 2)
```

Comment:
I  think what you are saying photon is x86_64 and dup2 will be there but let me remind you we can cross compile it by just setting GOOS and if you set it to arm64 (for some reason) this will fail :)

It's not a must change but I thought we could just use dup3 while at it :)